### ISSUE TITLE
TransformerXL: is this really an issue of too high learning rate ?

### ISSUE THREAD

**Issue by kasparlund** [fastai/fastai#1736]
fastai version: '1.0.46.dev0'

nb trainingtokens: 2.5e9
drop_mult=0
configuration: standard TransformerXL
%time learn.fit_one_cycle(cyc_len=2, max_lr=5e-4, moms=(0.8,0.7), wd=1e-3)

Total time: 110:29:25
epoch	train_loss	valid_loss	accuracy
1	2.760716	2.756633	0.442434
2	6.228361	6.220452	0.076458

![image](https://user-images.githubusercontent.com/679479/53693240-edc85600-3d9d-11e9-8ada-14163f9e6450.png)

Total number of iterations is 831117
Lowest training loss occure at epoch , loss = 513568, 2.6658912 
at epoch 515530 the loss increases to 6 within 150 iterations

**Comment by sgugger** [comment]
It may also be the moms too low that breaks training. I had no issue training a model on WT103 to a very low perplexity.
I'd suggest keeping the conversation on the forum until we identify a clear bug (if there is one) and then open an issue when that's the case.
